Earthfile

```diff
@@ -22,6 +22,7 @@ ARG GOLINT_VERSION=1.47.3
 ARG GO_VERSION=1.18
 
 all:
+  BUILD +webui-deps
```

Comment:
better to connect it to the target that builds the agent instead of untied to +all. Something like this:

```
build-kairos-agent: 
        COPY +webui-deps/node_modules ./internal/webui/public
        DO ....
```